Description: Fixes the compatibility with the version of the MongoDB driver in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/log4j-mongodb3/src/main/java/org/apache/logging/log4j/mongodb3/MongoDbProvider.java
+++ b/log4j-mongodb3/src/main/java/org/apache/logging/log4j/mongodb3/MongoDbProvider.java
@@ -172,7 +172,7 @@
                 if (Strings.isNotEmpty(userName) && Strings.isNotEmpty(password)) {
                     description += ", username=" + userName + ", passwordHash="
                             + NameUtil.md5(password + MongoDbProvider.class.getName());
-                    mongoCredential = MongoCredential.createCredential(userName, databaseName, password.toCharArray());
+                    mongoCredential = MongoCredential.createMongoCRCredential(userName, databaseName, password.toCharArray());
                 }
                 try {
                     final int portInt = TypeConverters.convert(port, int.class, DEFAULT_PORT);
